### ISSUE TITLE
Constant refreshing on HTC One S 

### ISSUE THREAD

**Issue by Alcaste** [HabitRPG/habitica#495]
When browsing the site it constantly refreshes, which prevents log-in or navigating.

Using default browser on the phone. Android version 4.0.3

**Comment by horusofoz** [comment]
This is definitely weird behaviour. Do you get similar behaviour else where? Can you try using an alternate browser and advise on the outcome?

**Comment by Alcaste** [comment]
Odd, I check it in response to this and now it seems to be working fine. I did the usual (different browser, empty browser caches, restart device) and it didn't seem to work at the time, though today it seems fine. I'll follow up if it happens again. It appeared to do it a few times before logging in, but after managing to log in it hasn't auto-refreshed.

**Comment by lefnire** [comment]
It happened to me too, I'll keep my eyes pealed while i'm in the code to
see what could be the cause.

**Comment by weelillad** [comment]
Thanks, Pandoro, for referencing my issue. Here's my experience with this bug.

I'm using Dolphin Browser, on a Samsung Galaxy S2 running a custom build of Ice Cream Sandwich (Slim ICS 4.0.4). When I'm connected to a Wifi network, and access the HabitRPG page, I encounter the same problem as Alcaste: The page loads completely, then within 2 seconds, the "Offline - Reconnect" notification appears, and the page refreshes itself. Any updates I manage to make, before the page refreshes, are not reflected.

Next, I switch off my Wifi connection and repeat the test on a new browser instance. Now, I am on the 3G cellular data network. The page still constantly refreshes itself. 

Finally, I repeat my experiment on the 2G data network. The page simply never finishes loading the first time. The data never stops flowing, though. I gave it 5 minutes before giving up for the sake of my wallet. This is different from my earlier experience documented in issue #334.

In summary, for me, HabitRPG cannot be used from a mobile phone.

Hope this helps somehow.

**Comment by paglias** [comment]
Hi! The site has been totally rewritten from the ground up, I'm closing the issue since it's probably fixed. Check if it's still happening and in case reopen the bug, thanks!
